src/Util/ClassSourceManipulator.php

```diff
@@ -185,7 +185,13 @@ public function addAccessorMethod(string $propertyName, string $methodName, $ret
 
     public function addGetter(string $propertyName, $returnType, bool $isReturnTypeNullable, array $commentLines = [])
     {
-        $methodName = 'get'.Str::asCamelCase($propertyName);
+        if ('bool' === $returnType) {
+            $snakeCasedPropertyName = Str::asSnakeCase($propertyName);
+            $propertyNameWithoutPrefix = (0 === strpos($snakeCasedPropertyName, 'is_')) ? substr($snakeCasedPropertyName, 3) : $propertyName;
```

Comment:
This logic seems wrong to me. We must always add either `get` or `is` as prefix on the property name. Otherwise, we are making this code incompatible with PropertyAccess (causing issues with the usage of Form and other components).

Reply:
That's sad ... because, if I'm right, it's generally recommended to add `is` to methods but never to property names. Couldn't we improve PropertyAccess instead? Thanks.

Reply:
Well, that case is trying to handle the case where the property itself starts with `is`, by skipping the prefix in such case. That's precisely what PropertyAccess does not support. If you insist on having `is` in the property name, PropertyAccess will expect `isIsFoo` or `getIsFoo` (or `hasIsFoo`).

MakerBundle can choose one of them, but should not generate `isFoo()` as getter.

If your naming follows your recommendation, you are precisely *not* relying on the weird magic that I'm asking to remove.

Reply:
As a matter of fact, I agree with @stof as it is more consistent. This is why the handling of `is` in property names is in [a second commit](https://github.com/symfony/maker-bundle/pull/456/commits/505d4fd2a19d209822e424435f0aa75bc2fea7bb). 

I chose to handle property names prefixed with `is` when I saw these two lines:
https://github.com/symfony/maker-bundle/blob/c4388410e2fb6321e77c5dd6e3cb2dba821f9fe6/src/Maker/MakeEntity.php#L314-L315

I thought that the `MakerBundle` would _always_ expect people to prefix boolean properties with `is`. Therefore, it made sense to me to handle this case.